src/core/meshcop/secure_transport.hpp

```diff
@@ -280,8 +297,108 @@ class Dtls : public InstanceLocator
     Error GetPeerCertificateBase64(unsigned char *aPeerCert, size_t *aCertLength, size_t aCertBufferSize);
 #endif // defined(MBEDTLS_BASE64_C) && defined(MBEDTLS_SSL_KEEP_PEER_CERTIFICATE)
 
+#if defined(MBEDTLS_SSL_KEEP_PEER_CERTIFICATE)
+    /**
+     * Returns an attribute value identified by its OID from the subject
+     * of the peer x509 certificate. The peer OID is provided in binary format.
+     * The attribute length is set if the attribute was successfully read or zero
+     * if unsuccessful. The ANS1 type as is set as defineded in the ITU-T X.690 standard
+     * if the attribute was successfully read.
+     *
+     * @param[in]     aOid                A pointer to the OID to be found.
+     * @param[in]     aOidLength          The length of the OID.
+     * @param[out]    aAttributeBuffer    A pointer to the attribute buffer.
+     * @param[in,out] aAttributeLength    On input, the size the max size of @p aAttributeBuffer.
+     *                                    On output, the length of the attribute written to the buffer.
+     * @param[out]    aAns1Type           A pointer to the ANS1 type of the attribute written to the buffer.
```

Comment:
I think we should use the string "Asn1" for var/function names, and use the fully correct spelling "ASN.1" in comments just to be clear on what standard we refer to.

Example below:

```suggestion
     * @param[out]    aAsn1Type           A pointer to the ASN.1 type of the attribute written to the buffer.
```

This can be applied to all lines that contain "Ans".

Reply:
fixed.